ext_emconf.php

```diff
@@ -8,11 +8,11 @@
     'uploadfolder' => false,
     'createDirs' => '',
     'clearCacheOnLoad' => false,
-    'version' => '8.1.1',
+    'version' => '10.0.0',
```

Comment:
I would prefer if we could make this a minor release which keeps compatibility with v8.
As far as I can see only the `Environment` class is the "deal breaker" here, right?

Reply:
My goal is to make a v10 compatible code. I do not have time allocated for anything else, sorry. So I can't work on yet another base.